pyproject.toml

```diff
@@ -141,22 +140,24 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 before-build = "bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
 
+[tool.cibuildwheel.linux.environment]
+# /project will be the $PWD equivalent inside the docker used to build the wheel
+PKG_CONFIG_PATH="/project/"
+
 [tool.cibuildwheel.macos]
-before-build = "bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+before-build = "pip install ninja; bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+
+[tool.cibuildwheel.macos.environment]
+# On macos-arm64, use Accelerate instead of setting this
+# CIBW_CONFIG_SETTINGS=\"setup-args=-Dblas=accelerate
```

Comment:
What is happening with this comment?

Reply:
Deleted, it was left over from some previous iteration of wheel build settings